src/lib/trick-o-matic.js

```diff
@@ -139,4 +141,26 @@ define([ 'jquery' ], function( $ ) {
     s.textContent = keyframes;
     $('svg', svg).prepend(s);
   };
-}); // end define
\ No newline at end of file
+  
+
+  
+}); // end define
+
+function trickomatic_initGAs() {
+  var src = this.src;
+  jQuery.ajaxSetup({async:false});
+  $.get(this.src, function(svg) { 
+    if( !svg ) return;  
+    // pipe-o-matic:
+    var pipes = svg.getElementsByClassName('pipe_group');
+    $(pipes).each(function(idx, pipe) {
+      var activeValues = $(pipe).data('cometvisu-active');
+      if ( !activeValues ) { return true; }
+      $(activeValues.split(' ')).each(function(idx, ga) {
+        templateEngine.addAddress(ga, 'svg');
+      });
+    });    
+  });
+  jQuery.ajaxSetup({async:true});
+};
```

Comment:
Do I interpret that correctly:
You are reloading the SVG backdrop just to parse it for the GAs? And are using a synchronous call for that?

If that's true I'm sorry to tell that this are two no gos for me:
- the data is already loaded and parsed in the browser and should be accessed through the DOM
- synchronous is considered bad practise in JavaScript and only very special occasions allow it